src/main/java/org/kuali/kfs/module/ar/document/CashControlDocument.java

```diff
@@ -0,0 +1,536 @@
+        return electronicPaymentClaims;
+    }
+
+    @Deprecated
+    public void setElectronicPaymentClaims(List<ElectronicPaymentClaim> electronicPaymentClaims) {
+        this.electronicPaymentClaims = electronicPaymentClaims;
+    }
+
+    public Document getReferenceFinancialDocument() {
+        DocumentService documentService = SpringContext.getBean(DocumentService.class);
+        Document document = null;
+        try {
+            document = documentService.getByDocumentHeaderId(getReferenceFinancialDocumentNumber());
+        } catch (WorkflowException we) {
+            LOG.warn("Unable to retrieve reference financial document: " + getReferenceFinancialDocumentNumber(), we);
+        }
+        return document;
+    }
+
+    public String getBankCode() {
+        return bankCode;
+    }
+
+    public void setBankCode(String bankCode) {
+        this.bankCode = bankCode;
+    }
+
+    /**
+     * Answers true when document payment medium is WIRE transfer.
+     */
+    @Override
+    public boolean answerSplitNodeQuestion(String nodeName) throws UnsupportedOperationException {
+        if (NODE_ASSOCIATED_WITH_ELECTRONIC_PAYMENT.equals(nodeName)) {
+            return ArConstants.PaymentMediumCode.WIRE_TRANSFER.equals(getCustomerPaymentMediumCode());
+        }
+        return super.answerSplitNodeQuestion(nodeName);
+    }
+
+    /**
+     * This is a helper method added to support workflow attribute configuration. This method helps to avoid attribute
+     * name mismatch between ProcessingChartOfAccountCode and chartOfAccountsCode.
+     *
+     * @return ProcessingChartOfAccountCode
+     */
+    public String getChartOfAccountsCode() {
+        if (getAccountsReceivableDocumentHeader() != null) {
+            return getAccountsReceivableDocumentHeader().getProcessingChartOfAccountCode();
+        }
+        return null;
+    }
+
+    /**
+     * This is a helper method added to support workflow attribute configuration. This method helps to avoid attribute
+     * name mismatch between ProcessingOrganizationCode and organizationCode.
+     *
+     * @return ProcessingOrganizationCode
+     */
+    public String getOrganizationCode() {
+        if (getAccountsReceivableDocumentHeader() != null) {
+            return getAccountsReceivableDocumentHeader().getProcessingOrganizationCode();
+        }
+        return null;
+    }
+
+    public Bank getBank() {
+        return bank;
+    }
+
+    public void setBank(Bank bank) {
+        this.bank = bank;
+    }
+
+    public void recalculateTotals() {
+        KualiDecimal total = KualiDecimal.ZERO;
+        for (CashControlDetail cashControlDetail : getCashControlDetails()) {
+            total = total.add(cashControlDetail.getFinancialDocumentLineAmount());
+        }
+        cashControlTotalAmount = total;
+        getFinancialSystemDocumentHeader().setFinancialDocumentTotalAmount(total);
+
+        // backport FINP-7430
```

Comment:
Just to confirm, are you backporting FINP-7340 or FINP-7430? Some of this overlay's comments mention one while some mention the other.

Reply:
fixing this now

Reply:
comment has been fixed to reference the correct FINP